tests/ext/test_typehandler.py

```diff
@@ -35,6 +35,16 @@ def test_slot_behaviour(self):
             assert getattr(inst, attr, "err") != "err", f"got extra slot '{attr}'"
         assert len(mro_slots(inst)) == len(set(mro_slots(inst))), "duplicate slot"
 
+    def test_repr(self):
+        # This function tests behaviour inherited from BaseHandler.
+        # TODO should this test be repeated in all other subclasses of BaseHandler except for
```

Comment:
see above

Reply:
Yes, moved the test from test_typehandler.py to test_basehandler.py (with necessary amendments)